static/js/src/entry/donate/Thermometer.vue

```diff
@@ -0,0 +1,61 @@
+<template>
+  <div class="thermometer">
+    <div class="bar">
+      <div class="bar_inner bar_inner--goal"></div>
+      <div class="bar_inner bar_inner--actual" v-bind:style="{width: actualWidth + '%'}"></div>
+    </div>
+    <div class="text">
+      <strong>{{ data[0].label }} new members</strong>
+      <br>
+      towards goal of {{ data[1].label }}.
+    </div>
+  </div>
+</template>
+
+<script>
+import axios from 'axios';
+
+export default {
+  name: 'Thermometer',
+
+  data() {
+    return {
+      loading: true,
+      error: false,
+      data: [
+        { slug: 'actual', label: '0', value: 0 },
+        { slug: 'goal', label: '0', value: 0 },
+      ],
+    };
+  },
+  computed: {
+    actualWidth: function() {
+      return ((this.data[0].value / this.data[1].value) * 100);
```

Comment:
Should this logic ensure the width never exceeds 100%? Looks like it's not causing CSS overflow if it's above 100%, but it's possible some browsers might interpret it differently. Probably not -- I'm just being paranoid.

Reply:
Amanda would love this paranoia! I hope we exceed the goal too :) 

To account for that, in the CSS I set the div to overflow: hidden.